code/modules/clothing/under/accessories/armor.dm

```diff
@@ -94,11 +98,13 @@
 		energy = ARMOR_ENERGY_SMALL,
 		bomb = ARMOR_BOMB_PADDED,
 	)
+	slowdown = 0.3
 
 /obj/item/clothing/accessory/leg_guard/heavy/scc
 	name = "heavy SCC leg guards"
 	icon_state = "legguards_blue"
 	item_state = "legguards_blue"
+	slowdown = 0
```

Comment:
Supposed to be 0?

Reply:
i kept the 0 slowdown on all the heavy SCC armour pieces as right now they are admin spawn only and have only been used in maybe one or two events. Though I'll probably change them all to match the other sets incase they see usage again.

Reply:
Made the SCC heavy set have slowdown for the sake of consistency incase the armour set is made available in future.